doc/source/whatsnew/v1.2.0.rst

```diff
@@ -547,6 +547,17 @@ ExtensionArray
 - Fixed an inconsistency in :class:`PeriodArray`'s ``__init__`` signature to those of :class:`DatetimeArray` and :class:`TimedeltaArray` (:issue:`37289`)
 - Reductions for :class:`BooleanArray`, :class:`Categorical`, :class:`DatetimeArray`, :class:`FloatingArray`, :class:`IntegerArray`, :class:`PeriodArray`, :class:`TimedeltaArray`, and :class:`PandasArray` are now keyword-only methods (:issue:`37541`)
 
+ToLatex
+^^^^^^^
+
+- Can now substitute simple fstrings in place of callable functions in to_latex
+
+.. ipython:: python
+    data = [[1, 2, 3], [4, 5, 6], [7, 8, 9.001]]
```

Comment:
Missing space here may be causing the CI failure. Also I think this should probably be in the I/O section.

Reply:
Got it thanks!